### DISCUSSION TITLE
Possible Issues with SimpleTheme?

### DISCUSSION THREAD

**Discussion by kuiperzone** [AvaloniaUI/Avalonia#10781] [Q&A] [answered]
I've got an application which I've updated for A11P6. Currently the app runs with FluentTheme, however, I've tried it with the new SimpleTheme. I'm running Linux/Gnome.

Shown below are two screenshots, the first is with SimpleTheme. The second is the same view, but with the app running FluentTheme (it's current default).

Possible issue with SimpleTheme:

It seems to draw borders around some controls that FluentTheme does not. Ie. around Buttons and ProjectTree.

SimpleTheme:

![image](https://user-images.githubusercontent.com/17156238/227506979-615cc1d2-18c1-4048-b94a-72961ce34cf3.png)

FluentTheme:

![image](https://user-images.githubusercontent.com/17156238/227507082-e4c10877-a53c-4351-b74e-69be9f1f9b28.png)

Also, would ask: FluentTheme.DensityStyle seems to do nothing? Is this correct?

**Describe the bug**

**To Reproduce**
You can get the source to my app:

https://github.com/kuiperzone/AvantGarde

Main branch currently builds out of the box on A11P6.

**Expected behavior**
Switching themes should not generally affects apps or show unusual artifacts unless the apps are overriding in some way.

**Screenshots**

**Desktop (please complete the following information):**

- OS: Fedora 37 / Gnome
- Version 11P6

**Additional context**

**Comment by timunie** [comment, marked as answer]
SimpleTheme and FluentTheme are complete different themes. So differnt outcome is expected. Check also ControlCatalog for reference. 

You can do the following to inspect the root cause: 
1. Look up the original styles in source
2. Use DevTools [F12] to inspect the visual tree. You should be able to see the borders which are bugging you. 

One more note: Denistiy styles only adjusts some margins and hights iirc. I also suggest to look it up in source.

**Comment by kuiperzone** [comment]
> SimpleTheme and FluentTheme are complete different themes. So differnt outcome is expected.

Thank you Tim.

Yes, I accept the reasoning here.
